test/instrumentation/modules/@aws-sdk/client-s3.test.js

```diff
@@ -130,7 +130,7 @@ tape.test('simple S3 V3 usage scenario', function (t) {
               cloud: { region: 'us-east-2' },
               service: { type: '', name: '', resource: 's3' }
             },
-            http: { status_code: 200 }
+            http: { status_code: 200, response: { encoded_body_size: 222 } }
```

Comment:
Review note: Changes to `encoded_body_size` in these tests are due to the localstack update which slightly changed the response XML for S3. See https://github.com/elastic/apm-agent-nodejs/pull/3334#issuecomment-1590196688 for an example.